stripe/test/resources/test_ephemeral_keys.py

```diff
@@ -0,0 +1,57 @@
+import stripe
+from stripe.test.helper import StripeResourceTest
+
+
+class EphemeralKeyTest(StripeResourceTest):
+
+    def test_create(self):
+        stripe.EphemeralKey.create(customer='cus_123',
+                                   api_version='2017-05-25')
+
+        self.requestor_class_mock.assert_called_with(
+            None,
+            api_version='2017-05-25',
+            account=None
+        )
```

Comment:
Rather than exposing `request_class_mock` here, what do you think about testing for the presence of the `Stripe-Version` header in the mock below? I think that should be enough? Here's an example of someone doing  a check on idempotency key:

``` python
        self.requestor_mock.request.assert_called_with(
            'post',
            '/v1/charges/ch_update_id/dispute/close',
            {},
            {'Idempotency-Key': 'foo'},
        )
```

I think generally it's preferable to keep the interface as small as possible so that these tests are mostly consistent in case we want to refactor all of them at some point in the future.